Source/Core/Core/HW/SystemTimers.cpp

```diff
@@ -102,6 +102,10 @@ static void DSPCallback(u64 userdata, s64 cyclesLate)
 static void AudioDMACallback(u64 userdata, s64 cyclesLate)
 {
 	int period = s_cpu_core_clock / (AudioInterface::GetAIDSampleRate() * 4 / 32);
+	if (SConfig::GetInstance().bEmulationSpeedFixAudio &&
```

Comment:
Oh, please do not touch the AI. AI is for emulation. I think you want to disable this: https://github.com/dolphin-emu/dolphin/blob/master/Source/Core/AudioCommon/Mixer.cpp#L62